### PR TITLE
Monsters chase Papercut around.

### PR DESCRIPTION
**Pull request by skylarjhdownes**
Stuff. Things.

### CODE REVIEW

**Comment by skylarjhdownes** [comment]
It doesn't like the selfie maker :(

**Comment by skylarjhdownes** [comment]
Reviews plox?  @ehgoodenough

File: source/scripts/models/Monster.js [hunk 1]
```diff
@@ -15,14 +15,25 @@ Monster.prototype.getStyle = function() {
...
+        this.y += 1 * delta
+    else 
+        this.y -= 1 * delta
+    if (window.game.ninja.x > this.x)
+        this.x += 1 * delta
+    else 
+        this.x -= 1 * delta
 }
 
+Monster.prototype.getPosition
```
Comment (ehgoodenough) [inline source/scripts/models/Monster.js:36]: This isn't finished just yet; I assume you'll finish later?

File: source/scripts/models/Monster.js [hunk 2]
```diff
@@ -15,14 +15,25 @@ Monster.prototype.getStyle = function() {
...
+
+    var chase = 0
+    if (window.game.ninja.y > this.y)
+        this.y += 1 * delta
+    else 
+        this.y -= 1 * delta
+    if (window.game.ninja.x > this.x)
+        this.x += 1 * delta
+    else 
+        this.x -= 1 * delta
```
Comment (ehgoodenough) [inline source/scripts/models/Monster.js]: Eventually, it'd be cool to change 1 from a magic number to `this.speed`

**Pull request merged**